api-reference/10 UI Components/Errors and Warnings/W1018.md

```diff
@@ -0,0 +1,20 @@
+---
+id: ErrorsUIWidgets.W1018
+---
+---
+##### shortDescription
+Occurs in the DataGrid when infinite scrolling is used and **selection**.[allowSelectAll](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#allowSelectAll) is set to **true**.
+
+---
+To select all rows, the component requires their total number. With infinite scrolling, the component cannot calculate it because rows are loaded dynamically.
+ 
+You can do one of the following to get rid of this warning:
```

Comment:
Since this section is called "Errors and Warnings", instead of "to get rid of this warning" can we say "to fix this error"?

Reply:
"To fix this, you can do one of the following:" is better. What about that?